back/engines/free/email_campaigns/app/policies/email_campaigns/campaign_policy.rb

```diff
@@ -16,7 +16,7 @@ def resolve
         elsif user&.active? && user&.project_moderator?
           projects = Project.where(id: user.moderatable_project_ids)
           if projects.any? { |p| p.visible_to == 'public' }
```

Comment:
Doesn't this mean we still could have the bug for moderators of private projects?

Reply:
By private, you mean when visibility limited to group(s)?

I'm not entirely sure, but I thought my changes in the following `else` block (where I remove the line that filters campaigns to only manual campaigns, would include all automated campaigns).